src/discord/commands/manage.ts

```diff
@@ -0,0 +1,166 @@
+import { ButtonInteraction, CommandInteraction, MessageEmbed, MessageActionRow, MessageButton, Message, SelectMenuInteraction } from 'discord.js';
+import Timeblock from '../../models/timeBlock.model';
+
+function updateEmbed(userArray: string[], maxCapacity: number, manageState: string) {
+    const embedTitle = manageState === 'add' ? 'Adding' : 'Removing';
+    const preposition = manageState === 'add' ? 'to' : 'from';
+
+    const embed = new MessageEmbed()
+        .setColor('#48d7fb')
+        .setTitle(`Currently ${embedTitle} Collaborators`)
+        .setDescription(
+            `To ${manageState} collaborators ${preposition} your booking, @ their Discord username!\n\nCurrently Invited Collaborators (${
+                maxCapacity! - userArray!.length
+            } available spaces left):\n<@!${userArray?.join('>\n<@!')}>`
```

Comment:
```suggestion
            } available spaces left):\n${userArray.map(user => `<@!${user}>`).join('\n')}`
```

I think this is a bit cleaner